javascript/react/src/components/Codebook/CodebookFieldEditor.styles.ts

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import styled from '@essex/styled-components'
+import type { ITheme } from '@fluentui/react'
+import { Label } from '@fluentui/react'
+
+export const FieldContainer = styled.div``
+export const OutsideLabel = styled(Label)`
+	position: absolute;
+	left: 3px;
+	margin-top: 10px;
+`
+
+export const Container = styled.div`
```

Comment:
General styling note: the fields definitely need some padding, which is apparent in the stats block and in single-field view where the labels bump right into the borders. I'd use the default padding we use on the verbs.